src/lwc/modal/modal.js-meta.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<LightningComponentBundle xmlns="http://soap.sforce.com/2006/04/metadata" fqn="modal">
+    <apiVersion>49.0</apiVersion>
```

Comment:
@bethbrains What would be a good label/description to put into this component that isn't specific to the adoption pages?

Reply:
@force2b it's not exposed so it doesn't need a label/descrip. Is it NPSP's practice to include them regardless? It would serve as dev docs only.

Reply:
We do try to include them, even as dev docs. In this case, other teams wouldn't necessarily know that this exists as a shared component that can be used outside of the getting started page context.

Reply:
Got it. Then I'd say label is `Generic Modal` and description could be `A generic, reusable modal component that follows SLDS since there is no lightning base component for this yet.` Does that sound ok to you?

Reply:
Thank you @bethbrains 
FYI @dcaballeroSf